src/Manipulations.php

```diff
@@ -555,6 +562,43 @@ public function apply()
         return $this;
     }
 
+    /**
+     * Create new manipulations class
+     *
+     * @return self
+     */
+    public static function create()
+    {
+        return new self();
+    }
+
+    /**
+     * Return manipulationSequence as array
+     *
+     * @return array
+     */
+    public function toArray()
+    {
+        return $this->manipulationSequence->toArray();
+    }
+
+    /**
+     * Checks if the given manipulations has arrays inside or not
+     *
+     * @param  Array $manipulations
+     * @return bool
+     */
+    private function checkIfHasMultipleConversions($manipulations)
```

Comment:
rename to `hasMultipleConverions` and add an `array` typehint and `bool` return type hint